x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/action/ReloadSynonymAnalyzerTests.java

```diff
@@ -143,20 +207,15 @@ public void testUpdateableSynonymsRejectedAtIndexTime() throws FileNotFoundExcep
 
         final String indexName = "test";
         final String analyzerName = "my_synonym_analyzer";
-        MapperException ex = expectThrows(MapperException.class, () -> client().admin().indices().prepareCreate(indexName)
-                .setSettings(Settings.builder()
-                .put("index.number_of_shards", 5)
-                .put("index.number_of_replicas", 0)
-                .put("analysis.analyzer." + analyzerName + ".tokenizer", "standard")
-                .putList("analysis.analyzer." + analyzerName + ".filter", "lowercase", "synonym_filter")
-                .put("analysis.filter.synonym_filter.type", "synonym")
-                .put("analysis.filter.synonym_filter.updateable", "true")
-                .put("analysis.filter.synonym_filter.synonyms_path", synonymsFileName))
-                .addMapping("_doc", "field", "type=text,analyzer=" + analyzerName).get());
-
-        assertEquals(
-                "Failed to parse mapping: analyzer [my_synonym_analyzer] "
-                + "contains filters [synonym_filter] that are not allowed to run in all mode.",
-                ex.getMessage());
+        MapperException ex = expectThrows(MapperException.class,
```

Comment:
This compression of multiple settings onto single lines seems to me to make test more difficult to follow, can we keep the indentation as it was before?

Reply:
Good point, this was not intended, must be the new Eclipse installation doing some autoformatting.